pkg/plugins/optional/helm/v1alpha/scaffolds/init.go

```diff
@@ -255,7 +278,7 @@ func (s *initScaffolder) copyConfigFiles() error {
 
 // copyFileWithHelmLogic reads the source file, modifies the content for Helm, applies patches
 // to spec.conversion if applicable, and writes it to the destination
-func copyFileWithHelmLogic(srcFile, destFile, subDir, projectName string) error {
+func copyFileWithHelmLogic(srcFile, destFile, subDir, namePrefix string) error {
```

Comment:
HI @mkarlheim

Are there any other manifests that we should append this prefix and that we are missing?